client/src/components/Button.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const Button = ({ title, type, myFunction }) => {
+  return (
+    <DefaultButton type={type} onClick={myFunction}>
+      {title}
+    </DefaultButton>
+  );
+};
+
+export default Button;
+
+const DefaultButton = styled.button`
+  background-color: inherit;
+  border: none;
+  background-color: var(--color-cyan);
```

Comment:
You override `background-color: inherit;` with `background-color: var(--color-cyan);`. You should decide which value you want. :)